src/tests/Feature/ProfileTest.php

```diff
@@ -250,4 +250,37 @@ public function it_should_save_user_course_application_and_send_whatsapp_group_j
         );
         $this->assertTrue($user->hasRole(Str::ucfirst($availableCourse->type)));
     }
+
+    /** @test */
+    public function it_should_return_user_courses()
+    {
+        $user = User::factory()->create();
+
+        $availableCourse = Course::factory()->available()->create();
+
+        $userExistingCourse = Course::factory()
+            ->available()
+            ->create(['type' => $availableCourse->type, 'is_active' => true]);
+
```

Comment:
böyle  available ve existing ayrı ayrı şartlara gerek yok bu servis için. 
user'ın tüm kurslarını döneceğiz index servisi olduğu için. 
aşağıdaki gibi bişey işimizi görecektir.  
Eğer bunu yaparsan aşağıdaki  foreach içinde $user_courses  ->  $userCourses  şeklinde güncellemek gerekecek.

```suggestion
        $userCourses = Course::factory()->count(rand(1, 3))->create();
```